lib/Cake/Controller/Controller.php

```diff
@@ -45,14 +45,21 @@
  *
  * @package       Cake.Controller
  * @property      AclComponent $Acl
+ * @property      string $action The action handling the current request. Deprecated, use CakeRequest::$action instead.
  * @property      AuthComponent $Auth
+ * @property      string $base Base URL path. Deprecated, use CakeRequest::$base instead.
  * @property      CookieComponent $Cookie
+ * @property      array $data POST data. Deprecated, use CakeRequest::$data instead.
  * @property      EmailComponent $Email
  * @property      PaginatorComponent $Paginator
+ * @property      array $params Array of parameters parsed from the URL. Deprecated, use CakeRequest::$params instead.
+ * @property      string $here The full address to the current request. Deprecated, use CakeRequest::$here instead.
```

Comment:
Sort properties alphabetically (`$here` and `$Flash` is out of order). And maybe list components first (also it will look better as they has no description here), then your additions.